File: check_git.py
import subprocess

try:
    res = subprocess.run(["git", "status"], capture_output=True, text=True, cwd="c:\\Users\\user\\3D Objects\\Portfolio")
    print("STATUS OUTPUT:")
    print(res.stdout)
    print(res.stderr)
    
    res2 = subprocess.run(["git", "ls-files", "client/public/logos"], capture_output=True, text=True, cwd="c:\\Users\\user\\3D Objects\\Portfolio")
    print("LS-FILES OUTPUT:")
    print(res2.stdout)
except Exception as e:
    print("Error:", e)

File: check_git.js
import { execSync } from 'child_process';
try {
  const output = execSync('git ls-files client/public/logos', { cwd: 'c:\\Users\\user\\3D Objects\\Portfolio' });
  console.log('TRACKED FILES:');
  console.log(output.toString());
} catch (e) {
  console.error('Error running git:', e.message);
  if (e.stderr) console.error(e.stderr.toString());
}
